docs/blog/_posts/2019-05-24-15th-dotty-milestone-release.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.15.0-RC1 – the fully bootstrapped compiler
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2019-05-24
+---
+
+Hi! In this article, we'd like to announce the 15th release of Dotty. The most exciting thing in this release is the full bootstrap for Dotty introduced by PR [#5923](https://github.com/lampepfl/dotty/pull/5923). This means that we are able to compile Dotty with Dotty itself, and hence use all the new features in the compiler code base.
+
+With this release comes a bunch of new features and improvements, such as the ability to enforce whether an operator is intended to be used in an infix position, the type safe pattern bindings and more.
+
+This release serves as a technology preview that demonstrates new
+language features and the compiler supporting them.
+
+Dotty is the project name for technologies that are being considered for
+inclusion in Scala 3. Scala has pioneered the fusion of object-oriented and
+functional programming in a typed setting. Scala 3 will be a big step towards
+realising the full potential of these ideas. Its main objectives are to
+
+- become more opinionated by promoting programming idioms we found to work well,
+- simplify where possible,
+- eliminate inconsistencies and surprising behaviours,
+- build on strong foundations to ensure the design hangs together well,
+- consolidate language constructs to improve the language’s consistency, safety, ergonomics, and
+  performance.
+
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our 15th scheduled release according to our
+[6-week release schedule](https://dotty.epfl.ch/docs/contributing/release.html).
+
+# What’s new in the 0.15.0-RC1 technology preview?
+## Operator Rules
+This change addresses the problem of the regulation of whether an operator is supposed to be used in an infix position. The motivation is for the library authors to be able to enforce whether a method or a type is supposed to be used in an infix position by the users. This ability will help to make code bases more consistent in the way the calls to methods are performed.
```

Comment:
This change addresses ... position. -> We introduce two annotations that regulate the use of operators. The first is called `@infix` and the second is called `@alpha`. The motivation behind `@infix` is ...

Reply:
Actually only one annotation, `@alpha` doesn't work yet (although is defined). #5975 says it will come in a separate PR, which I haven't seen yet.

Reply:
@odersky since `@alpha` is not yet in I propose to enclose in comments the relevant section in the docs.